packages/sit-onyx/src/components/OnyxToast/types.ts

```diff
@@ -0,0 +1,34 @@
+import type { DensityProp } from "../../composables/density";
+import type { OnyxColor } from "../../types";
+
+export type OnyxToastProps = DensityProp & {
+  /**
+   * Main toast headline.
+   */
+  headline: string;
+  /**
+   * Optional toast description (e.g. detailed error message).
+   */
+  description?: string;
+  /**
+   * Toast color.
+   */
+  color?: Extract<OnyxColor, "neutral" | "danger" | "warning" | "success">;
+  /**
+   * Duration in milliseconds for the toast to close automatically.
+   * Timer will be paused when hovering the toast.
+   *
+   * Can be set to `0` to disable the auto closing.
```

Comment:
```suggestion
   * Can be set to `0` to disable the auto closing.
   *
   * @default 5000
```

Reply:
The default value is already documented by Storybook, should I still add this?

![image](https://github.com/SchwarzIT/onyx/assets/67898185/b188ab11-aadb-4682-90f7-05b0b54b9518)